reframe/core/schedulers/torque.py

```diff
@@ -61,10 +61,12 @@ def _update_state(self, job):
             raise JobError('qstat failed: %s' % completed.stderr, job.jobid)
 
         nodelist_match = re.search(
-            r'exec_host = (?P<nodespec>\S+)', completed.stdout
+             re.compile(r'exec_host = (?P<nodespec>[\S\t\n]+)', re.MULTILINE),
```

Comment:
You don't need `re.compile` here. The `search()` function accepts flags already, so you can pass the `re.MULTILINE` directly.